src/parser.cc

```diff
@@ -232,6 +232,26 @@ static void insertIncludedElement(sdf::SDFPtr _includeSDF,
   }
 
   // Validate included model's frame semantics
+  if (!_config.CustomModelParsers().empty())
+  {
+    // Since we have custom parsers, we can't create a throwaway sdf::Root
+    // object to validate the merge-included model. This is because calling
+    // `sdf::Root::Load` here would call the custom parsers if this model
+    // contains a nested model that is custom parsed. But the custom parsers
+    // will be called again later when we construct the final `sdf::Root`
+    // object. We also can't do the merge here since we'd be doing so without
+    // validating the model.
+    // We could forego validating the model and just merge all it's children to
```

Comment:
```suggestion
    // We could forego validating the model and just merge all its children to
```

Reply:
Done in https://github.com/gazebosim/sdformat/pull/1293/commits/b4c352287333e85b69fa5ce96cc185a5cd9d006b